packages/@vuepress/plugin-search/src/client/composables/useHotKeys.ts

```diff
@@ -10,7 +10,7 @@ export const useHotKeys = ({
 }): void => {
   const onKeydown = (event: KeyboardEvent): void => {
     if (!input.value || hotKeys.value.length === 0) return
-    if (event.target === document.body && hotKeys.value.includes(event.key)) {
+    if (hotKeys.value.includes(event.key)) {
```

Comment:
Will this be triggered when user typing `s` in the search input?

Reply:
Maybe, even not, I will suggest adding `Ctrl` key. I will update this PR. Opening it is just for having a place to ask the above question.